src/components/Navigator/BaseNavigatorCard.vue

```diff
@@ -44,43 +34,33 @@
 
 <script>
 import {
-  INDEX_ROOT_KEY,
   SIDEBAR_HIDE_BUTTON_ID,
 } from 'docc-render/constants/sidebar';
-import SidenavIcon from 'theme/components/Icons/SidenavIcon.vue';
-import Reference from 'docc-render/components/ContentNode/Reference.vue';
-import Badge from 'docc-render/components/Badge.vue';
+import InlineCloseIcon from 'theme/components/Icons/InlineCloseIcon.vue';
 import { baseNavOpenSidenavButtonId } from 'docc-render/constants/nav';
 
 export default {
   name: 'BaseNavigatorCard',
   components: {
-    SidenavIcon,
-    Reference,
-    Badge,
+    InlineCloseIcon,
   },
   props: {
```

Comment:
Doesn't seem like this component needs any props after the refactor

Reply:
I deleted them here: https://github.com/apple/swift-docc-render/pull/772/commits/44bbbe1762a785f6e75730bcaaa99d5c4bdcc80d